ngcompiler/lib/v1/src/compiler/attribute_matcher.dart

```diff
@@ -9,7 +9,7 @@ abstract class AttributeMatcher {
   final String name;
   final String? value;
 
-  const AttributeMatcher(this.name, this.value);
+  const AttributeMatcher(this.name, [this.value]);
```

Comment:
Why is this necessary?

Reply:
In order to resolve ugly null parameter passing to super() below on line 81, by making second nullable parameter as optional:
class SetAttributeMatcher extends AttributeMatcher {  
--     SetAttributeMatcher(String name) : super(name, null);
++   SetAttributeMatcher(super.name);